index.js

```diff
@@ -1,21 +1,20 @@
-require('dotenv').config()
-
-const fs = require('fs');
-const Discord = require('discord.js');
-const Client = require('./client/Client');
-const config = require('./config.json');
-const {Player} = require('discord-player');
-
-const {ActivityType} = require('discord.js');
-
+import dotenv from 'dotenv';
+dotenv.config();
+
+import fs from 'fs';
+import Discord, {ActivityType}  from 'discord.js';
+import Client from './client/Client.js';
+import config from './config.json' assert { type: "json" };
+import { Player } from 'discord-player';
 const client = new Client();
 client.commands = new Discord.Collection();
 
 const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));
 
 for (const file of commandFiles) {
-    const command = require(`./commands/${file}`);
+  import(`./commands/${file}`).then((command) => {
```

Comment:
The way you use import here, you will get the following:
```Javascript
[Module: null prototype] {
  default: {
    name: 'volume',
    description: 'Change the volume!',
    options: [ [Object] ],
    execute: [AsyncFunction: execute]
  }
}
```

Instead of:

```Javascript
{
  name: 'volume',
  description: 'Change the volume!',
  options: [
    {
      name: 'volume',
      type: 4,
      description: 'Number between 0-200',
      required: true
    }
  ],
  execute: [AsyncFunction: execute]
}
```

The `console.log` on line 21 will also display an empty array instead of the command. To fix this we can change it to:

```Javascript
for (const file of commandFiles) {
  const command = await import(`./commands/${file}`);
  client.commands.set(command.default.name, command.default);
}
```